imap_processing/cdf/config/imap_idex_l2b_variable_attrs.yaml

```
@@ -72,6 +85,30 @@ impact_charge_bins:
     UNITS: " "
     DICT_KEY: SPASE>Support>SupportQuantity:Other
 
+rectangular_lon_pixel:
+  CATDESC: Longitude pixel index for IDEX SkyMap
```

If these are in units of degrees I think you should remove "index", maybe indicate it is the "center" of the pixel.
```suggestion
  CATDESC: Longitude pixel center for IDEX SkyMap
```